webhook/resources/volume/mutator.go

```diff
@@ -99,20 +99,22 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 
 	moreLabels := map[string]string{}
 	size := volume.Spec.Size
+	backupTargetName := volume.Spec.BackupTargetName
 	if volume.Spec.FromBackup != "" {
-		bName, bvName, _, err := backupstore.DecodeBackupURL(volume.Spec.FromBackup)
+		bName, remoteBVName, _, err := backupstore.DecodeBackupURL(volume.Spec.FromBackup)
```

Comment:
What's purpose of the renaming? What does `remoteBVName` mean?

Reply:
Because the `BackupVolume` name in the cluster is changed to be a random UUID with a prefix, renaming the volume name on the remote backup target should be reasonable. WDYT?